app/helpers/application_helper/button/cockpit_console.rb

```diff
@@ -13,4 +14,8 @@ def on?
     return @record.current_state == 'on' if @record.respond_to?(:current_state) # VM status
     @record.ready_condition_status == 'True' if @record.respond_to?(:ready_condition_status) # Container status
   end
+
+  def platform_supported?
+    @record.platform.downcase != 'windows' if @record.respond_to?(:current_state)
```

Comment:
I don't quite understand how `:current_state` and `@record.platform` come together.

As in if we don't know the current state, we also do not know the platform? I'm not saying this
is incorrect, I'm just curious.

Also, if the above is correct, I'd rather change the above line to:
```ruby
@record.respond_to?(:current_state) && @record.platform.downcase != 'windows'
```